specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/examples/ListManagedInstanceOperations.json

```diff
@@ -0,0 +1,72 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "sqlcrudtest-7398",
+    "managedInstanceName": "sqlcrudtest-4645",
+    "api-version": "2018-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-7398/providers/Microsoft.Sql/managedInstances/sqlcrudtest-4645/operations/11111111-1111-1111-1111-111111111111",
+            "name": "11111111-1111-1111-1111-111111111111",
+            "type": "Microsoft.Sql/managedInstances/operations",
+            "properties": {
+              "managedInstanceName": "sqlcrudtest-4645",
+              "operation": "UpsertManagedServer",
+              "operationFriendlyName": "MANAGED SERVER UPDATE",
+              "percentComplete": 0,
+              "startTime": "2017-06-01T09:10:08.10Z",
+              "state": "InProgress"
+            }
+          },
+          {
+            "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-7398/providers/Microsoft.Sql/managedInstances/sqlcrudtest-4645/operations/22222222-2222-2222-2222-222222222222",
```

Comment:
>/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-7398/providers/Microsoft.Sql/managedInstances/sqlcrudtest-4645/operations/22222222-2222-2222-2222-222222222222 [](start = 19, length = 189)

You seem to be returning IDs in the list call, but there's no declared path that lets you GET a single operation by that ID.

Reply:
Yes. In ManagedInstanceOperations (and ServerOperations). By calling this API. the operations on server and MI will be returned. Fetching operation status by ID is currently not supported.